.eslintrc.js

```diff
@@ -0,0 +1,188 @@
+module.exports = {
```

Comment:
That's a lot of "offs"! 

I set up [this config](https://github.com/damassi/gatsby-starter-typescript-rebass-netlifycms/blob/master/.eslintrc.js) on a personal project which matches things pretty well (in the sense that most code I write is in Reaction, and so that's the defaults I'm most familiar with). Might be something here?